mtls-northbound/docs/cloudshell-tutorial.md

```diff
@@ -0,0 +1,586 @@
+    --location=global
+```
+
+### Strict
+Create the strict policy yaml file.
+```
+cat << EOF > ${ROOT}-strict.yaml
+name: ${ROOT}-strict
+mtlsPolicy:
+  clientValidationMode: REJECT_INVALID
+  clientValidationTrustConfig: projects/${PROJECT}/locations/global/trustConfigs/${TRUST_CONFIG}
+EOF
+```
+Create the strict policy.
+```
+gcloud beta network-security server-tls-policies import ${ROOT}-strict \
+    --source=${ROOT}-strict.yaml \
+    --location=global
+```
+
+## Prepare Configuration files to Update the GLB
+The steps are the same for any Application GLB, just find the Target HTTPS Proxy for the GLB and update.
+You can easily switch between the TLS policies by creating representations for each configuration.
+
+Find the Target HTTPS Proxy for your GLB.
+```
+gcloud compute target-https-proxies list
+```
+Example response:
+```
+                                SSL_CERTIFICATES            URL_MAP              REGION    CERTIFICATE_MAP
+apigee-proxy-https-proxy        gm-xapi-kurtkanaskie-net    apigee-proxy-url-map
+apigee-proxy-modern-https-proxy gm-m-xapi-kurtkanaskie-net  apigee-proxy-modern-url-map
+```
+
+### Create Target HTTPS Proxy configurations
+We'll create multiple configurations so we can easily switch between them, one for "none", "lenient" and "strict".
+
+Set the TARGET_PROXY env variable
+```
+export TARGET_PROXY=apigee-proxy-https-proxy
+```
+
+Export the Target HTTPS Proxy configuration and store in a "none" file.
+This allows us to remove the Server TLS Policy later.
+```
+gcloud compute target-https-proxies export ${TARGET_PROXY} \
+   --global \
+   --destination=${TARGET_PROXY}-none.yaml
+```
+Copy to create new files for "lenient" and "strict" and add the appropriate Server TLS Policy names.
+
+Lenient
+```
+cp ${TARGET_PROXY}-none.yaml ${TARGET_PROXY}-lenient.yaml
+
+echo "serverTlsPolicy: //networksecurity.googleapis.com/projects/${PROJECT}/locations/global/serverTlsPolicies/${ROOT}-lenient" >> ${TARGET_PROXY}-lenient.yaml
+```
+
+Strict
+```
+cp ${TARGET_PROXY}-none.yaml ${TARGET_PROXY}-strict.yaml
+
+echo "serverTlsPolicy: //networksecurity.googleapis.com/projects/${PROJECT}/locations/global/serverTlsPolicies/${ROOT}-strict" >> ${TARGET_PROXY}-strict.yaml
+```
+
+## Update the GLB Backend Service with mTLS headers
+First, find the Backend Service(s) for the GLB.
+The custom header configuration must be added to each Backend Service, in this example we only have one.
+```
+gcloud compute backend-services list --global
+```
+
+Example response:
+```
+NAME                  BACKENDS                                                                                PROTOCOL
+apigee-proxy-backend  us-east1/instanceGroups/apigee-mig-us-east1,us-west1/instanceGroups/apigee-mig-us-west1 HTTPS
+```
+
+Set the BACKEND_SERVICE env variable
+```
+export BACKEND_SERVICE=apigee-proxy-backend
+```
+[Update the custom headers](https://cloud.google.com/load-balancing/docs/https/custom-headers-global#mtls-variables).
```

Comment:
Alter the hyperlink to only encompass `[custom headers]`

Reply:
OK